### DISCUSSION TITLE
pbmcMultiome Dataset not available through SeuratData as described in Integrating scRNA-seq and scATAC-seq data Vignette?

### DISCUSSION THREAD

**Discussion by jdparker101** [satijalab/seurat#4042] [Q&A] [answered]
Hi, I'm working through some of the vignettes for Seurat to familiarise myself with the package and I'm currently on Integrating scRNA-seq and scATAC-seq data. However, the pbmcMultiome data described in the vignette does not appear to be part of Seurat data, even though the vignette claims it is. I just get a dataset not available error when I run the code as described in the vignette and the dataset is not present when I run AvailableData(). Am I missing something? Any help would be much appreciated.

Jacob

**Comment by andrewwbutler** [comment, marked as answer]
Hi Jacob,

Thanks for bringing this up. I think we've resolved the issue and the data should be available now if you try to view/install.
